chainregistry.go

```diff
@@ -751,3 +760,43 @@ func initNeutrinoBackend(chainDir string) (*neutrino.ChainService, func(), error
 
 	return neutrinoCS, cleanUp, nil
 }
+
+func parseHeaderStateAssertion(state map[string]string) (
+	*headerfs.FilterHeader, error) {
+
+	if len(state) == 0 || state == nil {
+		return nil, nil
+	}
+
+	var headerStateAssertion = &headerfs.FilterHeader{}
+	heightArg, ok := state["height"]
+	if !ok {
+		return nil, fmt.Errorf("unable to create neutrino " +
+			"light client: required parameter height " +
+			"missing from header state assertion")
```

Comment:
Nit: the error here shouldn't contain context of callers, so we should drop `unable to create...` from the error string.